builtin/logical/rabbitmq/backend.go

```diff
@@ -0,0 +1,118 @@
+package rabbitmq
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+	"github.com/michaelklishin/rabbit-hole"
+)
+
+// Factory creates and configures Backends
+func Factory(conf *logical.BackendConfig) (logical.Backend, error) {
+	return Backend().Setup(conf)
+}
+
+// Backend creates a new Backend
+func Backend() *framework.Backend {
+	var b backend
+	b.Backend = &framework.Backend{
+		Help: strings.TrimSpace(backendHelp),
+
+		PathsSpecial: &logical.Paths{
+			Root: []string{
+				"config/*",
+			},
+		},
+
+		Paths: []*framework.Path{
+			pathConfigConnection(&b),
+			pathConfigLease(&b),
+			pathRoles(&b),
```

Comment:
Can you consider adding `pathListRoles` as well, just like `postgresql` backend?